lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -84,6 +89,7 @@ class CircleImageBase extends NodeBase {
       this.width = width;
       this.height = height;
       this.radius = 0.5 * this.width;
+      this.refreshNeeded = false;
```

Comment:
OK, I understand what is happening; `needsRefresh()` is/should be called in the child classes. So the changes I made to `CircleImageBase` are misplaced.

`needsRefresh()` is already called in `Circle.resize()`. For completeness, the call should also be used in `CircularImage.resize()`.

I will make an update to the PR which I hope will clarify the point. Well spotted, @mojoaxel.